Hardware/Hardware/CardReader/StripeCardReader/Charge+Stripe.swift

```diff
@@ -4,12 +4,29 @@ extension Charge {
 
     /// Convenience initializer
     /// - Parameter charge: An instance of a StripeTerminal.SCPCharge
-    init(charge: SCPCharge) {
+    init(charge: StripeCharge) {
         self.id = charge.stripeId
-        self.amount = Int(charge.amount)
+        self.amount = charge.amount
         self.currency = charge.currency
         self.status = ChargeStatus.with(status: charge.status)
-        self.description = charge.description
+        self.description = charge.stripeDescription
         self.metadata = charge.metadata
     }
 }
+
+/// The initializers of SCPCharge are annotated as NS_UNAVAILABLE
+/// So we can not create instances of that class in our tests.
+/// A workaround is declaring this protocol, which matches the parts of
+/// SCPCharge that we are interested in, make Crahge implement it,
```

Comment:
nit:

```suggestion
/// SCPCharge that we are interested in, make Charge implement it,
```

I also like these protocols to surface the properties we care about 👍